modules/ingestor/src/graph/vulnerability/mod.rs

```diff
@@ -28,6 +29,7 @@ impl VulnerabilityInformation {
             || self.published.is_some()
             || self.modified.is_some()
             || self.withdrawn.is_some()
+            || self.cwe.is_some()
```

Comment:
Yeah, I start to wonder if we should pass Option<FooInfo> instead of always FooInfo possibly composes of nothing but `None`.

Not your problem; I did this.